lib/gemstash/http_client.rb

```diff
@@ -14,7 +14,13 @@ def initialize(message, code)
   end
 
   #:nodoc:
+  class ConnectionError < StandardError
```

Comment:
Makes sense, I don't think we need to handle that case any different than the web error anyway.

I was just making the logic explicit in this first step.